maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -1759,27 +1757,8 @@ static void populateProperties( CommandLine commandLine, Properties systemProper
         systemProperties.setProperty( "maven.build.version", mavenBuildVersion );
     }
 
-    private static void setCliProperty( String property, Properties properties )
+    private static void setCliProperty( String name, String value, Properties properties )
     {
-        String name;
-
-        String value;
-
-        int i = property.indexOf( '=' );
-
-        if ( i <= 0 )
-        {
-            name = property.trim();
-
-            value = "true";
-        }
-        else
-        {
-            name = property.substring( 0, i ).trim();
-
-            value = property.substring( i + 1 );
-        }
-
         properties.setProperty( name, value );
```

Comment:
Since there is only one line left, does it make sense to keep this method at all if it is not used anywhere else?

Reply:
I'm fine with inlining it. I initially kept it as-is because of the lengthy comment in there; I can't really judge whether it's (still) relevant.

Reply:
Right, I have discussed this with @rfscholte. This will go away in 4.0.0. Keep as-is for now.